### PR TITLE
lang: spelling and typography fixes for French

### PR DESCRIPTION
**Pull request by Glandos**
Fix spelling and typography

### CODE REVIEW

File: app/i18n/lang/fr.json [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
   "help": "Aide",
-  "termsLink": "Conditions d'utilisation",
+  "termsLink": "Conditions d’utilisation",
   "privacyLink": "Politique de confidentialité",
   "sendFeedbackLink": "Envoyer des commentaires",
-  "aboutLink": "A propos",
+  "aboutLink": "À propos",
   "sourceLink": "Code source",
-  "versionLabel": "Version: {{version}}",
+  "versionLabel": "Version : {{version}}",
```
Comment (saghul) [inline app/i18n/lang/fr.json:10]: There should be no space before the colon.

**Review by saghul — DISMISSED** [review]

File: app/i18n/lang/fr.json (hunk 1 above)
Comment (Glandos) [reply app/i18n/lang/fr.json:10]: https://fr.wikipedia.org/wiki/Espace_ins%C3%A9cable#En_France

> Le deux-points et les guillemets en chevrons constituent une exception en ce que le code typographique français recommande d'insérer devant le deux-points, ou entre les guillemets et le texte qu'ils renferment, non une espace fine insécable mais une espace insécable (espace-mot insécable).

It should even be a non-breaking space. I grew up in this typographic culture, and honestly, I find it difficult to read when there is no space before the colon :smile:

Comment (csett86) [reply app/i18n/lang/fr.json:10]: Thank you for highlighting and fixing this

**Review by csett86 — APPROVED** [review]

**Comment by csett86** [comment]
The build failure is unrelated to your commit. I accidentally broke mac ci for PRs from other repositories in #697, reverted in #700

**Comment by csett86** [comment]
@Glandos, can you please rebase to current master? Then mac ci build should be working again.

**Review by csett86 — APPROVED** [review]

**Comment by csett86** [comment]
Thank you for your contribution @Glandos

**Pull request merged**